planty/planty/settings.py

```diff
@@ -10,10 +10,10 @@
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
 # flake8: noqa
-
-from pathlib import Path
 from django.contrib.messages import constants as messages
 import os
+from pathlib import Path
```

Comment:
These two imports are from standard library so you can move them at the very top